test/kubernetes/e2e/features/helm/types.go

```diff
@@ -18,4 +18,13 @@ var (
 			},
 		},
 	}
+
+	enterpriseCRDCategory = "solo-io"
+	CommonCRDCategory     = "gloo-gateway"
+
+	enterpriseCRDs = []string{
```

Comment:
nit: It could be useful to callout that we have other CRDs that have a mix of APIs (some open source, some enterprise) and that we don't consider those "enterprise CRDs" since they contain open source functionality

Reply:
The new README hopefully clarifies a bit, but not sure where you want this called out and if that's the right place.